src/components/panes/CallPane.scss

```diff
@@ -1,5 +1,9 @@
 .CallPane.PaneBase {
     min-width: 500px;
+
+    @include small-screen {
+        min-width: 100%;
```

Comment:
Could be set from PaneBase by being more specific. By this, there is no need to type this in all Panes that is wider by any reason.

In PaneBase.scss, when setting the small-screen styling, we can write:
`&[class] { min-width: auto; }`

This would target all cases where we have ".CallPane.PaneBase" and such.

Reply:
Fixed!